### ISSUE TITLE
사용자인증(5) - 장고(Django) 핥짝 맛보기

### ISSUE THREAD

**Issue by swarf00** [swarf00/swarf00.github.io#14]
https://swarf00.github.io/2019/09/06/associate-user.html 

 장고ORM 의 ForeignKey 사용법을 배워봅니다. ForeignKey는 다소 많은 옵션을 제공하지만 자주 사용되는 옵션은 몇가지 되지 않으니 소개해드리는 것만은 꼭 기억하시길 바랍니다.

**Comment by MinkyungPark** [comment]
안녕하세요 좋은 글 정말 감사합니다. 이대로 쭉 따라했는데, createsuperuser로 슈퍼유저를 생성하였는데 그 계정으로 어드민에 접속이 안됩니다 ㅜㅜ 쿼리문으로 is_superuser 값에 1을 주고 해봐도 로그인이 안되는데 어떤 문제일까요?

**Comment by swarf00** [comment]
> @MinkyungPark
> 안녕하세요 좋은 글 정말 감사합니다. 이대로 쭉 따라했는데, createsuperuser로 슈퍼유저를 생성하였는데 그 계정으로 어드민에 접속이 안됩니다 ㅜㅜ 쿼리문으로 is_superuser 값에 1을 주고 해봐도 로그인이 안되는데 어떤 문제일까요?

뭔가 설정의 문제가 있을 것으로 보이는데, 오류 메시지를 올려주시면 도와드릴 수 있을 것 같습니다.

**Comment by MinkyungPark** [comment]
> @swarf00
> > @MinkyungPark
> > 안녕하세요 좋은 글 정말 감사합니다. 이대로 쭉 따라했는데, createsuperuser로 슈퍼유저를 생성하였는데 그 계정으로 어드민에 접속이 안됩니다 ㅜㅜ 쿼리문으로 is_superuser 값에 1을 주고 해봐도 로그인이 안되는데 어떤 문제일까요?
> 
> 뭔가 설정의 문제가 있을 것으로 보이는데, 오류 메시지를 올려주시면 도와드릴 수 있을 것 같습니다.

쿼리문으로 is_active, is_staff모두 값을 1로 주었더니 로그인이 되었습니다! 커스텀유저모델이 뭔가 잘못된 것 같습니다. 해결해보도록하겠습니다. 장고로 처음 웹 짜보는데 이 블로그시리즈가 정말 많은 도움이 되었습니다 감사합니다!

**Comment by swarf00** [comment]
예 별 내용이 없는데 도움이 돼서 다행입니다. 요즘 바빠서 글을 더 쓸 여유가 없네요. 시간이 나면 좀 더 재밌는 주제들을 소개해드릴께요.
